### DISCUSSION TITLE
ILS G/S

### DISCUSSION THREAD

**Discussion by LukasGNN** [flybywiresim/aircraft#5347] [Help] [answered]
Hi.

i recently experience problems with G/S in ILS. 
The plane manage to follow the centerline but when it should start descent to RWY (follow the G/S) nothing happens.
I have checked that ILS i tuned, i have latest dev. On 3 attampts to land on 3 different airports, 3 times G/S was not be followed.
Anyone experiencing the same?
![bilde](https://user-images.githubusercontent.com/46678717/126074916-b67941ef-96a6-4813-a154-0e1b89f362f1.png)

BR
Lukas

**Comment by ExampleWasTaken** [comment, marked as answer]
The FMA needs to show at least `LOC*` in order to capture the G/S.
